src/frameworks/foundation/ns_string.rs

```diff
@@ -933,7 +933,10 @@ pub fn get_static_str(env: &mut Environment, from: &'static str) -> id {
     if let Some(&existing) = State::get(env).static_str_pool.get(from) {
         existing
     } else {
-        let new = msg_class![env; _touchHLE_NSString_Static alloc];
+        // Using allocWithZone: instead of alloc: allows us to avoid having to leave alloc: as
+        // unchecked.
```

Comment:
Could you explain what the type error here, or in `alloc`, is? (By the way, there's no colon at the end of its name, it has no argument.) It's not obvious to me what would be wrong.

Reply:
I should probably have specified, it's because this gets called during late linking, and the stack pointer is set to zero at that point. If you call `alloc:`, it dips into NSObject, so i'd have to set that as unchecked as well.

Reply:
Ahh. The stack setup should probably be done earlier then.